curvefs/src/client/s3/disk_cache_write.cpp

```diff
@@ -303,15 +305,20 @@ int DiskCacheWrite::AsyncUploadRun() {
 }
 
 int DiskCacheWrite::AsyncUploadStop() {
+    if (isRunning_.load()) {
+        std::unique_lock<std::mutex> lock(mtx_);
+        while (!waitUpload_.empty()) {
```

Comment:
we should wait the objs that has been swapped(waitUpload_  swap to toUpload in GetUploadFile) all to be uploaded.

Reply:
oops, I will fix it.